roadmap/implementers-guide/src/runtime/router.md

```diff
@@ -90,26 +92,28 @@ struct HrmpChannel {
     sender_deposit: Balance,
     /// The amount that the recipient supplied as a deposit when accepting opening this channel.
     recipient_deposit: Balance,
+    /// The maximum message size that could be put into the channel.
+    limit_message_size: u32,
```

Comment:
```suggestion
```

If, as I suspect, this is a redundant bit of data, then we can just remove it.

Reply:
Your suspicion is correct! This PR undergone quite a few of rebases and [git] fixups and I guess this is one artifact of that.